api/tests/acceptance/complete-login.test.js

```diff
@@ -0,0 +1,77 @@
+const { Canteen } = require('../../infrastructure/models/canteen');
+const { LoginToken } = require('../../infrastructure/models/login-token');
+const { User } = require('../../infrastructure/models/user');
+const { createUserWithCanteen } = require('../../infrastructure/repositories/user');
+const { init } = require('../../server');
+const { sequelize } = require('../../infrastructure/postgres-database');
+const { NoLoginTokenError } = require('../../domain/errors');
+
+jest.mock('../../domain/usecases/complete-login');
+const { generateJWTokenForUser } = require('../../domain/usecases/complete-login');
```

Comment:
Pourquoi ce mock ? Dans un test d'acceptance on essaie d'avoir le minimum de mock, on va mocker que ce qui sort de l'appli (fetch par exemple).

Reply:
Oui - j'ai noté ça mais pas le corriger encore... je vais le faire !